### PR TITLE
[WIP] ci: install CircleCI

### PR DESCRIPTION
**Pull request by alban**
Also, avoid using `latest` tags: pick up a specific tag of traceloop (I picked https://github.com/kinvolk/traceloop/pull/20).

- Build URL: https://circleci.com/gh/kinvolk/inspektor-gadget
- Container image for the 'gadget' DaemonSet: https://cloud.docker.com/u/kinvolk/repository/docker/kinvolk/gadget/tags
- Build artifact for kubectl-gadget: available in the CircleCI page

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+version: 2
+jobs:
+ build:
+   docker:
+     - image: circleci/golang:1.12
```
Comment (invidian) [inline .circleci/config.yml]: Maybe go 1.13? :)

Comment (alban) [reply .circleci/config.yml]: ok :)

**Comment by alban** [comment]
> Why circle Ci? Maybe we could use github actions instead?

Just because I know CircleCI more, so I know how to do things. I am open to try github actions too, if it achieves the same result :)

The features I need in traceloop and inspektor-gadget are:
- build in different containers for different stages of the build
- build some parts with Dockerfile
  - with access to previously built files outside of the "docker build" command
  - run the resulting container image to build further artifacts
- push a built container image to a registry
- push a built artifact

**Comment by alban** [comment]
Superseded by https://github.com/kinvolk/inspektor-gadget/pull/25, and it uses GitHub Actions instead.

**Pull request closed without merging**